### PR TITLE
Revamp development tooling

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Creates two tasks, one `yarn dev` to develop at localhost:3000 without hot reloading and with the JS bundle being served through our server; and one `yarn dev-hot` using webpack-dev-server with hot reload.

### CODE REVIEW

**Review by frm — APPROVED** [review]
Tested and working 👍

File: README.md [hunk 1]
```diff
@@ -9,18 +9,19 @@ For dependencies:
...
 
-To run the server and the client: 
+To run the server and the client:
 ```
 yarn dev
 ```
-The script above will run the express server and the webpack-dev-server at the sime time. 
-The server is at `localhost:3000` but the client code is actually served at `localhost:8080`. 
-Use the second address to have access to hot module reloading.
+The script above will run the express server and webpack in watch mode at the sime time.
```
Comment (frm) [inline README.md]: *same

File: src/frontend/components/Firefighter/index.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 import "./index.css";
 
 const label = {
-  active: "disponível",
+  active: "Disponível",
```
Comment (frm) [inline src/frontend/components/Firefighter/index.js:7]: cheeky

**Comment by frm** [comment]
@jfranciscosousa already approved, feel free to merge

File: README.md (hunk 1 above)
Comment (jfranciscosousa) [reply README.md]: :+1:

**Pull request merged**